Library/Homebrew/test/language/perl/shebang_spec.rb

```diff
@@ -5,19 +5,12 @@
 require "utils/shebang"
 
 describe Language::Perl::Shebang do
-  let(:file) { Tempfile.new("perl-shebang") }
+  let(:file) { File.open("#{TEST_TMPDIR}/perl-shebang", "w") }
```

Comment:
```suggestion
  let(:file) { Pathname.new("perl-shebang") }
```
as this was technically a `Pathname` not `File`. Similarly, I think `TEST_TMPDIR` may be implicit.

Generally it's considered 👎🏻 to use `File.open` without the block form.

Reply:
We have `mktmpdir` which creates a directory is automatically cleaned after a test.